packages/SwingSet/docs/run-policy.md

```diff
@@ -88,7 +88,7 @@ while(1) {
   processInboundIO();
   const policy = make100CrankPolicy();
   await controller.run(policy);
-  commit();
+  await commit();
```

Comment:
Please also update callers of `commit()` in `packages/solo/src` and `packages/cosmic-swingset/src`.

Reply:
They were already `await`ing!